cmd/resultscollector/main.go

```diff
@@ -226,105 +233,187 @@ func getConfigMap(owner *unstructured.Unstructured, configMapName, filename stri
 			},
 		},
 		Data: map[string]string{
-			filename: string(contents),
+			"exit-code": exitcode,
+			filename:    string(contents),
 		},
 	}
 }
 
+func uploadToResultServer(arfContents *resultFileContents, scapresultsconf *scapresultsConfig) error {
+	return backoff.Retry(func() error {
+		url := scapresultsconf.ResultServerURI
+		fmt.Printf("Trying to upload to resultserver: %s\n", url)
+		reader := bytes.NewReader(arfContents.contents)
+		client := &http.Client{}
+		req, _ := http.NewRequest("POST", url, reader)
```

Comment:
Why do we ignore the error here?

Reply:
Cause this would only return an error in case we would set up an invalid url name, an invalid method or an invalid reader. And those are hardcoded values here.

The real request is done in line 254.